packages/chains/src/arbitrum/params.ts

```diff
@@ -10,6 +10,7 @@ const specificParams: ChainParams = {
     chainIdHex: "0xA4B1",
     name: "Arbitrum One",
     shortName: "Arbitrum",
+    uniswapV3Fee: 3000,
```

Comment:
should there be any configuration of the fee level for different assets/pairs?

Reply:
@vminkov @carlomazzaferro please check this update.

Reply:
Looks good now